src/phases/common-anim-phase.ts

```diff
@@ -1,14 +1,18 @@
 import type { BattlerIndex } from "#enums/battler-index";
 import type { CommonAnim } from "#enums/common-anim";
-import { CommonBattleAnim } from "#app/data/battle-anims";
+import { CommonBattleAnim } from "#app/data/battle-anims/common-battle-anim";
 import { PokemonPhase } from "#app/phases/abstract-pokemon-phase";
 import { globalScene } from "#app/global-scene";
+import { PhaseId } from "#enums/phase-id";
 
 /**
  * Plays a {@linkcode CommonBattleAnim}
  * @extends PokemonPhase
  */
 export class CommonAnimPhase extends PokemonPhase {
+  /** @override **Must** use generic {@linkcode PhaseId} because as {@linkcode CommonAnimPhase} is extended by other phases */
```

Comment:
```suggestion
  /** @override **Must** use generic {@linkcode PhaseId} since {@linkcode CommonAnimPhase} is extended by other phases */
```

Much clearer now thank you. "because as" is too many justifications, putting since instead but "because" or "as" would be fine too, just not both :p